src/Scripts/UI/Controls/GraphicsEditor.gd

```diff
@@ -41,7 +54,7 @@ var brush_color: Color:
 
 var _last_pos: Vector2
 var _draw_begin: bool = false
-
+var bubble: CloudControl = Buble.instantiate()
```

Comment:
we could change the variable name to bubble_inst or instance to avoid confusion with the scene variable